Services/Mail/PRIVACY.md

```diff
@@ -0,0 +1,82 @@
+# Mail Service Privacy
+
+This documentation does not warrant completeness or correctness. Please report any
+missing or wrong information using the [ILIAS issue tracker](https://mantis.ilias.de)
+or contribute a fix via [Pull Request](docs/development/contributing.md#pull-request-to-the-repositories).
+
+## General Information
+
+- An account with `Edit Settings` permissions for `Administration > User and Roles >
+  Roles > RoleTitle > Default Permissions > Administration Permission > Mail` can remove the
+  permission `Internal Mail: User can use internal mail system` for all users of that role.
+- The delivery to external email addresses via `SMTP` or `Sendmail` can be enabled and configured in the
+  global administration by an account with `Edit Settings` permissions for `Administration > Communication > Mail >
+  Settings`. External email delivery (concrete: using explicit email addresses as recipient) can be
+  also restricted to certain roles.
+- Consumers might use the `Mail Service` to send emails to external email addresses (directly) or use the
+  channelled message delivery of the internal API without considering any activation.
+  Please check the documentation of the respective consumer.
+
+## Data being stored
+
+- For each message created in the `Mail Service` the ID of the user account that created the message is stored
+  (table: `mail`) along with the entered message data like **subject**, **body**, **attachments**, **recipient strings**,
+  **placeholder information** and an additional **datetime** information. For every internal recipient (user account) of
+  this message an additional record is created as a copy. The `Mail Service` differentiates between
+  the `user_id` (means: `Owner`), and the `sender_id` (understandably means: `Sender`). For each copy,
+  the `user_id` is substituted by the ID of the respective recipient user account, so each user account owns its own record.
+- Until a user explicitly sends the message or created a draft, the message data is temporarily stored as a shadow copy
+  (without any datetime information) in database table `mail_saved` The purpose is to temporarily store the message 
+  data in case the user navigates to other screens in the `Mail Service` (e.g. `Search` or `Attachments` screens).
+- Files uploaded as potential attachments (the users' personal attachment stage) are stored in
+  the `<EXTERNAL_DATA_DIRECTORY>/mail` directory of the ILIAS installation, prefix with the user's ID.
+- Files eventually attached to a sent message are stored in the `<EXTERNAL_DATA_DIRECTORY>/mail/<PATH>[/<PATH>]`
+  subdirectories of the ILIAS installation, where the path is constructed from the message ID. Attached files are
+  shared among the message copies (table: `mail_attachment`).
```

Comment:
This passage is more important for IT security and the Art. 15 GDPR export, but should be deleted from Privacy.md.